### ISSUE TITLE
Locks the Chosen missions

### ISSUE THREAD

**Issue by Matamun** [long-war-2/lwotc#663]
Locks the Chosen missions behind the plot.
Specifically:
Revealing the Strongholds requires that either you have killed a Codex, or that you have completed the Black Site.
Unlocking the Stronghold Missions proper, requires either killing the damaged Avatar, or completing the Psi Gate Mission, or completing the Forge mission.
Copied: https://steamcommunity.com/sharedfiles/filedetails/?id=1140251687

**Comment by pledbrook** [comment]
Does that mod work with LWOTC?

**Comment by Matamun** [comment]
For me - no. I add it, because after i kill chosens, game is bit easily, and i am get chosens' weapons very early

**Comment by Fallguy55** [comment]
I agree that this would be a good move. I would like to see the first chosen only killable after killing the codex, the second one after the blacksite, and the third one after killing the avatar. Or possibly the third one not being killable at all

**Comment by Anxoczt** [comment]
I rather get them out ASAP and for good but for LW2 i think the soldiers rank should be the solution. Simply bump the ranks required so you need top of the line troops for the final piece.

**Comment by tunbridgep** [comment]
Might be good as a second wave option. "Killing chosen requires story progress".

This way, people (like me) who find the chosen extremely easy, would have options

Maybe a "Make chosen unkillable" second wave option two, which still lets you remove them from missions but never truly destroy them. After you kill them 5 times, you can research their weapons.
